src/components/shared/footer.tsx

```diff
@@ -82,6 +83,23 @@ interface FooterCcpaProps {
     isCcpa: boolean;
 }
 
-const FooterCcpa: FC<FooterCcpaProps> = ({ isCcpa }) => <Footer>{footerContents(isCcpa)}</Footer>
+const renderContent = (ccpaStatus: boolean): JSX.Element => {
+    if (!ccpaStatus){
```

Comment:
Is this either/or? or do we need to show both?

Reply:
It's also below, so could we see this twice?

Reply:
yes, that was the older version should have been ommited